neqo-transport/src/send_stream.rs

```diff
@@ -705,9 +716,13 @@ impl SendStreams {
         self.0.insert(id, stream);
     }
 
-    pub fn acked(&mut self, token: StreamRecoveryToken) {
+    pub fn acked(&mut self, token: &StreamRecoveryToken) {
         if let Some(ss) = self.0.get_mut(&token.id) {
-            ss.mark_as_acked(token.offset, token.length as usize, token.fin);
+            if let Ok(len) = token.length.try_into() {
+                ss.mark_as_acked(token.offset, len, token.fin);
+            } else {
+                //FIXME: Do we want to log something?
```

Comment:
Raising awareness wrt this path. It's never supposed to happen but well...

Reply:
Don't log anything, just use `unwrap()` and we can panic instead.  The number of layers this has to pass with both u64 and usize variables carrying the value is insane.  If it doesn't fit in both, we're in a very bad place.

The alternative is to change token.length to a usize.  That ends up requiring far fewer conversions overall.